setup/owncloud.sh

```diff
@@ -32,28 +28,31 @@ if [ ! -f $STORAGE_ROOT/owncloud/config.php ] \
 	ln -sf $STORAGE_ROOT/owncloud/config.php /usr/local/lib/owncloud/config/config.php
 fi
 
-# Check if ownCloud dir exist, and check if version matches owncloud_ver (if either doesn't - install/upgrade)
-if [ ! -d /usr/local/lib/owncloud/ ] \
-	|| ! grep -q $owncloud_ver /usr/local/lib/owncloud/version.php; then
+InstallOwncloud() {
+        version=$1
+        hash=$2
 
-	# Download and verify
-	wget_verify https://download.owncloud.org/community/owncloud-$owncloud_ver.zip $owncloud_hash /tmp/owncloud.zip
+	# Remove the current owncloud
+	rm -rf /usr/local/lib/owncloud
```

Comment:
The comment on the unzip has always been incorrect, previously the directory was moved. It can indeed be changed for clarity.